pkg/workloadmeta/collectors/podman/podman.go

```diff
@@ -154,6 +154,18 @@ func (c *collector) expiredEvents() []workloadmeta.CollectorEvent {
 	return res
 }
 
+func networkIPs(container *podman.Container) map[string]string {
+	res := make(map[string]string)
+
+	if len(container.State.NetworkStatus) > 0 {
+		if len(container.State.NetworkStatus[0].IPs) > 0 {
+			res["podman"] = container.State.NetworkStatus[0].IPs[0].Address.IP.String()
```

Comment:
any reason why we don't collect the IPs available?

Reply:
it would allow supporting `"%%host_<NETWORK NAME>%%"` - [docs](https://docs.datadoghq.com/agent/faq/template_variables/)

Reply:
Should be addressed by #10543.